File: template/java/snip-getter.java
/**
 * get {{_input_:name}}
 * @return {{_var_:name}}
 */
public {{_input_:type}} get{{_expr_:substitute('{{_var_:name}}', '\w\+', '\u\0', '')}}() {
	return {{_var_:name}};
}

File: template/php/snip-getter.php
/**
 * Get {{_input_:property}}
 *
 * @return {{_input_:type}}
 */
public function get{{_expr_:substitute('{{_var_:property}}', '\w\+', '\u\0', '')}}() {
	return $this->{{_var_:property}};
}
